packages/runtime/test/src/interpreter/inner-transaction.ts

```diff
@@ -1409,6 +1410,23 @@ describe("Inner Transactions", function () {
 			});
 		});
 
+	});
+
+	describe("Teal v7", function () {
+		describe("Inner transaction can run in v7", () => {
+			useFixture("teal-files");
+			this.beforeEach(() => {
+				setUpInterpreter(1, ALGORAND_ACCOUNT_MIN_BALANCE);
+			});
+			it("Should execute a teal version 7 file successful ", () => {
```

Comment:
Lets rephrase it. We already know we are executing tealv7 from the `describe` part. Lets say what it should execute. Should execute inner transaction successfully?